index.html

```diff
@@ -31,7 +31,7 @@ <h1 align="center"><b>
 </div>
 <br>
 <div class="container">
-   <div> <a href="Saad's_Portfolio/index.html"><img src="Pictures/saad-profilepic.png" alt="ProfilePicture-saad.png" title="ProfilePicture-saad.png"></a></div>
+   <div> <a href="https://www.tinyurl.com/mastigramofficial"><img src="Pictures/saad-profilepic.png" alt="ProfilePicture-saad.png" title="ProfilePicture-saad.png"></a></div>
```

Comment:
@Saad-ahamed-m Nice work. I could see the Instagram clone. Are you trying to showcase your project ? It would be better if we stick with the portfolio. What do you think ? 

`saad-ahamed-m.github.io` can have your portfolio
`saad-Ahamed-m.github.io/{project-name}` can be your project showcase.

Reply:
@yogeshvar I had done the necessary changes